packages/vector_graphics/lib/src/listener.dart

```diff
@@ -450,16 +450,33 @@ class FlutterVectorGraphicsListener extends VectorGraphicsCodecListener {
     double y,
     int fontWeight,
     double fontSize,
+    int decoration,
+    int decorationStyle,
+    int decorationColor,
     Float64List? transform,
     int id,
   ) {
+    final List<ui.TextDecoration> decorations = <ui.TextDecoration>[];
+    if (decoration & 0x1 != 0) {
+      decorations.add(ui.TextDecoration.underline);
+    }
+    if (decoration & 0x2 != 0) {
+      decorations.add(ui.TextDecoration.overline);
+    }
+    if (decoration & 0x4 != 0) {
+      decorations.add(ui.TextDecoration.lineThrough);
+    }
```

Comment:
Consider making these named constants

Reply:
I almost wish they were exposed from the `dart:ui` class but will do.

Reply:
Done